test/faker.spec.ts

```diff
@@ -41,7 +41,7 @@ describe('faker', () => {
   describe('rawDefinitions', () => {
     it('locale rawDefinition accessibility', () => {
       // Metadata
-      expect(faker.rawDefinitions.metadata.title).toBeDefined();
+      expect(faker.rawDefinitions.metadata!.title).toBeDefined();
```

Comment:
I mean this literally checks for the value being defined. No need to assert non-null.

```suggestion
      expect(faker.rawDefinitions.metadata?.title).toBeDefined();
```

Reply:
AFAICT the ! is not an assertion, its a not null definition expression.
Or am I wrong?

Reply:
The name of the operator is [Non-null assertion operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator). It does only assert in a TS context tho.

The problem is that `faker.rawDefinitions.metadata` might be `null` or `undefined`. This will propably never happen, but TS doesn't know that. In general you should not use the the NNA operator. See [@typescript-eslint/no-non-null-assertion](https://typescript-eslint.io/rules/no-non-null-assertion/)

Reply:
In the end it doesn't hurt to change this. Done.